Code/GraphMol/FileParsers/MolFileStereochem.h

```diff
@@ -82,5 +82,27 @@ RDKIT_FILEPARSERS_EXPORT void invertMolBlockWedgingInfo(ROMol &mol);
 RDKIT_FILEPARSERS_EXPORT void markUnspecifiedStereoAsUnknown(ROMol &mol,
                                                              int confId = -1);
 
+//! generate enhanced stereo groups based on the status of the chiral flag
+/// property
+/*
+ \param mol: molecule to be modified
+ \param zeroFlagGroupType: how to handle non-grouped stereo centers when the
+        chiral flag is set to zero
+
+  If the chiral flag is set to a value of 1 then all specified tetrahedral
+  chiral centers which are not already in StereoGroups will be added to an
+  ABS StereoGroup.
+
+  If the chiral flag is set to a value of 0 then all specified tetrahedral
+  chiral centers will be added to a StereoGroup of the type zeroFlagGroupType
+
+  If there is no chiral flag set (i.e. the property is not present), the
+  molecule will not be modified.
```

Comment:
The rest of the RDKit (at least the places I find) all use the logic that the chiral flag defaults to zero and that this only changes if the property is present. The usual pattern is:
```
  unsigned int chiralFlag = 0;
  mol.getPropIfPresent(RDKit::common_properties::_MolFileChiralFlag,
                       chiralFlag);
```
So at the moment we don't have anywhere in the code which treats missing as any different from zero. Since that kind of fits with the V2000 format (see below), I think it's a reasonable way to continue. Make sense?


As an aside: I just checked the docs: It looks like the chiral flag is not optional for V3000 and is always there for V2000 (if it's not actually present in the counts line then it defaults to zero).

Reply:
I just deleted a long comment which missed the point.
The code for when the chiral flag is set to zero will never be executed because the mol file reader doesn't set the property when it's zero.

This needs to be changed.
Thanks for catching that.

Reply:
No problem!

Reply:
Ok, I made the corresponding change to the mol file parsers and added a test.  Let's see what happens to the CI builds. In theory this shouldn't break anything. If it does then I'll need to write some docs about the change. :-)